packages/code-studio/src/styleguide/StyleGuide.test.tsx

```diff
@@ -24,3 +24,22 @@ describe('<StyleGuide /> mounts', () => {
     ).not.toThrow();
   });
 });
+
+describe('StyleGuide test mode', () => {
+  beforeEach(() => {
+    window.HTMLElement.prototype.scroll = jest.fn();
+    window.HTMLElement.prototype.scrollIntoView = jest.fn();
+  });
+
+  test('default does not activate the isolate input box', () => {
+    const { container } = render(
+      <ApiContext.Provider value={dh}>
+        <ThemeProvider themes={[]}>
+          <StyleGuide />
+        </ThemeProvider>
+      </ApiContext.Provider>
+    );
+
+    expect(container.querySelector('input[placeholder="Isolate"]')).toBeNull();
```

Comment:
I like that you are testing this. I would recommend using an `id` instead of depending on placeholder text since this is a special input that should only ever have 1 instance.

Reply:
Actually see suggestion regarding hashchange event handler below